### DISCUSSION TITLE
missing menue point for rejecting mounted USB-drive

### DISCUSSION THREAD

**Discussion by mic62** [mucommander/mucommander#638] [Q&A]
Could it be that it is not possible to savely unmount a connected USB-drive or SD-card from within muCommander? At least I couldn`t find such menue point.

**Comment by mic62** [comment]
I'm back to oMega as there was no answer for the last three weeks here. Seems the developer, after a great start with speedy support, now lost any interest in his app.

**Comment by ahadas** [comment]
Hi @mic62 , sorry for not getting back to you about that before. I actually looked for a cross-platform library that can do that but didn't find one that looked promising. We can initiate calls to platform tools per-operating system type but then we need to make sure those are built-in tools. If you specify the operating system you use, I can see if there's a reasonable solution for that and if not, you can consider adding a custom command that does it

**Reply by mic62** [reply]
Hello ahadas,
great to hear from you and thankyou for your response.
I'm running mucommander on Windows 11. Having no developer skills I'm not familar with what you wrote about libraries and platform calls.
